### PR TITLE
Only show series position if a book is in a series

### PR DESCRIPTION
**Pull request by amit1307**
## Summary of change

- Only show series position if a book is in a series.

- Check box to indicate if book is in a series.

## Related issue

https://github.com/knjk04/book-project/issues/170

### CODE REVIEW

**Comment by knjk04** [comment]
Thanks for this. As a UI change was involved, could you also add a screenshot of the book form?

**Comment by amit1307** [comment]
Default checkbox off and series position not visible:
image

When checkbox on:
image

**Comment by amit1307** [comment]
> Thanks for this. As a UI change was involved, could you also add a screenshot of the book form?

Done, thanks.

**Comment by knjk04** [comment]
Sorry, could you resolve the merge conflicts?

File: src/test/java/com/karankumar/bookproject/ui/book/BookFormTests.java [hunk 1]
```diff
@@ -106,29 +117,36 @@ private Book createBook(PredefinedShelfService predefinedShelfService) {
...
-        Assertions.assertEquals(firstName, bookForm.authorFirstName.getValue());
-        Assertions.assertEquals(lastName, bookForm.authorLastName.getValue());
-        Assertions.assertEquals(readShelf.getPredefinedShelfName(), bookForm.shelf.getValue());
-        Assertions.assertEquals(genre, bookForm.bookGenre.getValue());
-        Assertions.assertEquals(pagesRead, bookForm.pagesRead.getValue());
-        Assertions.assertEquals(numberOfPages, bookForm.numberOfPages.getValue());
-        Assertions.assertEquals(dateStarted, bookForm.dateStartedReading.getValue());
-        Assertions.assertEquals(dateFinished, bookForm.dateFinishedReading.getValue());
-        Assertions.assertEquals(rating, bookForm.rating.getValue());
+        assertEquals(bookTitle, bookForm.bookTitle.getValue());
```
Comment (knjk04) [inline src/test/java/com/karankumar/bookproject/ui/book/BookFormTests.java:139]: Could you change the asserts back to use the qualified class name (`Assertions.assertEquals`), as with your other PR?

**Comment by knjk04** [comment]
Thanks for this PR!

**Pull request merged**